mapboxgl/templates/main.html

```diff
@@ -359,6 +360,9 @@
     if (expressionType == 'match') {
         expression = generateMatchExpression(propertyValue, stops, defaultValue)
     }
+    else if (expressionType == 'identity') {
+        expression = ['get', propertyValue]
+    }
     else if (propertyValue == 'identity') {
```

Comment:
Is this line duplicative of line 363 now?

Reply:
Good q. I was trying to remember if there would still be a use case for both when I was looking into adding the "step" expression too.

Reply:
Addressed in 7335eb7.

Reply:
Addressed in 7335eb7d2bd521265303108f89ee95232caf2d52